reference/api/overview.md

```diff
@@ -1,45 +1,115 @@
 # API reference
 
-Welcome to the Meilisearch API documentation.
+Welcome to the Meilisearch API documentation. If you are new to Meilisearch, check out our [quick start guide!](/learn/getting_started/quick_start.md)
 
-::: tip
+Meilisearch is a RESTful API. This page describes the general behavior of the API.
 
-Check out [the FAQ](/resources/faq.md) for answers to some common questions 💡
+## OpenAPI
 
+The Meilisearch OpenAPI specifications:
+
+- [JSON](https://bump.sh/doc/meilisearch.json)
+- [YAML](https://bump.sh/doc/meilisearch.yaml)
+
+## Document conventions
+
+The API documentation uses the following conventions:
+
+- Curly braces (`{}`) represent path parameters, e.g., GET `/indexes/{index_uid}`
```

Comment:
I think we should use the comma as it's used everywhere else in the documentation. 
https://github.com/meilisearch/documentation/pull/1721

Reply:
I…never knew `e.g.,` was an actual thing and completely missed #1721. I'm against it because I always find it very awkward when two punctuation marks meet.  I mean, I know it's inevitable, but the less I see `",`, `—,`, `.,`, and so on, the happier I am. Will follow the majority, though.

Also, regarding `e.g.` specifically, I try not to use it a lot when writing for Meilisearch. [According to gov.uk](https://insidegovuk.blog.gov.uk/2016/07/20/changes-to-the-style-guide-no-more-eg-and-ie-etc/), `e.g.` can be difficult to  understand for people not super comfortable with English and screen readers don't always parse it correctly. They recommend using `for example`, `such as`, `like`, and `including` depending on the sentence.

Reply:
I agree we should get rid of it eventually and create a Vale rule so we use the full form instead.